pkg/webreg/webreg.go

```diff
@@ -734,15 +734,90 @@ syntax highlight all commands as bash.
 </p>
 
 <p>
-Sharing files between steps is supported via a shared directory. All
-containers will have an environment variable <code>SHARED_DIR</code> which
-contains the path of the shared directory. An artifacts directory also exists
-for steps that produce artifacts. The path for the artifacts directory is
-stored in the <code>ARTIFACTS_DIR</code> environment variable.
+A reference may be referred to in chains, workflows, and <code>ci-operator</code> configs.
 </p>
 
+<h4 id="execution"><a href="#execution">Step Execution Environment</a></h4>
 <p>
-A reference may be referred to in chains, workflows, and <code>ci-operator</code> configs.
+While a step simply defines a set of commands to run in a container image,
+by virtue of executing within a <code>ci-operator</code> workflow, the commands
+have a number of special considerations for their execution environment.
+The commands can expect a set of environment variables to exist that inform
+them of the context in which they run; commands in steps can communicate to
+other steps via a shared directory in their filesystem.
+</p>
+
+<h5 id="env"><a href="#env">Available Environment Variables</a></h5>
+<p>
+The following environment variables will be available to commands in a step: 
+</p>
+
+<table>
```

Comment:
This table has very little spacing between the columns on some rows. It might be a good idea to add `class=table` like the other tables in the UI to help space it out a bit more and make the styling consistent with the rest of the page.